support-frontend/assets/pages/contributions-landing/components/ContributionTypeTabs.jsx

```diff
@@ -60,67 +62,74 @@ const mapDispatchToProps = (dispatch: Function) => ({
 
 // ----- Render ----- //
 
+export const yellowChoiceCard = css`
+  &:checked + label {
+    background-color: #FFE500;
+    box-shadow: inset 0 0 0 4px #F3C100;
+  };
+  &:hover + label {
+    box-shadow: inset 0 0 0 4px #F3C100;
+  }
+`;
+
 function withProps(props: PropTypes) {
   const contributionTypes = props.contributionTypes[props.countryGroupId];
 
 
-  const renderContribTypeChoiceCards = () => (
-    <>
-      <ChoiceCardGroup
-        name="contributionTypes"
-        orientation="horizontal"
-      >
-        {contributionTypes.map((contributionTypeSetting: ContributionTypeSetting) => {
+  const renderYellowChoiceCards = () => (
```

Comment:
again, it looks like `renderYellowChoiceCards` and `renderControl` can be consolidated into
`const renderChoiceCards = (cssOverrides: SerializedStyles | null) => (...)`